### ISSUE TITLE
Allow deselecting zip code area

### ISSUE THREAD

**Issue by simzou** [hackforla/lucky-parking#296]
### Overview
Right now you have to unload the zip code layer to deselect it. 

### Action Items
We can make it so that double clicking the same zip deselects it or some other button.

### Resources/Instructions
@mg78856 suspects there's a simple fix to this, so assigning it to him.

**Comment by macho-catt** [comment]
Re-assigning this to @glenflorendo since this seems like a good first issue

**Comment by glenflorendo** [comment]
I have a solution, but I'm having issues testing since I'm not getting any data after clicking on a zip code area on the map.

```
GET http://localhost:5000/api/citation/draw/zip?zip=90064 404 (Not Found)
```

I delved deeper into the error, and I found that the database query for `controllers.zipSelect` isn't working. I get the following error:

```
error: relation "zip_4326" does not exist
```

I checked the database, and I also do not see this table.

I'm not sure if I'm doing something wrong, but any help is appreciated! 😄
